sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/AzureMonitorTransmitter.cs

```diff
@@ -100,17 +100,18 @@ private static ApplicationInsightsRestClient InitializeRestClient(AzureMonitorEx
             return new ApplicationInsightsRestClient(new ClientDiagnostics(options), pipeline, host: connectionVars.IngestionEndpoint);
         }
 
-        private static PersistentBlobProvider? InitializeOfflineStorage(AzureMonitorExporterOptions options, IPlatform platform)
+        private static PersistentBlobProvider? InitializeOfflineStorage(AzureMonitorExporterOptions options, IPlatform platform, string instrumentationKey)
```

Comment:
This class has a public property `InstrumentationKey`, should we use the property instead of passing the parameter?

Reply:
That would require changing the method from `static`. 
I would prefer these Initialize methods remain static to enable better unit testing.

Reply:
It does not make a difference as this is a private method and the initialization is called from constructor. In my opinion we should not create duplicates to help the tests, if that is the case the method or the class in not well-written.

Reply:
These Initialize methods are critically important because they're responsible for building all of the internals in our SDK and I want these to have very high test coverage.
Of the 4 Initialize methods, the only one that has 100% coverage is the one that's `internal static` and in my opinion, all 4 should move in this direction.
If all of these methods are instance based, then we can't test one without also testing all the others.

Reply:
Discussed offline. 
Will keep `static` to align with testing goals.
Will modify method parameters to eliminate duplication.